jupyter_theme_editor/handlers.py

```diff
@@ -0,0 +1,42 @@
+import json
+from jupyter_server.base.handlers import APIHandler
+from jupyter_server.utils import url_path_join
+import tornado
+from jinja2 import Template
+
+
+class RouteHandler(APIHandler):
+    # The following decorator should be present on all verb methods (head, get, post,
+    # patch, put, delete, options) to ensure only authorized user can request the
+    # Jupyter server
+
+    @tornado.web.authenticated
+    def post(self):
+        # input_data is a dictionary with a key "name"
+        input_data = self.get_json_body()
+        new_input_data = {}
+        for key, value in input_data.items():
+            value = str(input_data[key])
+            new_value = value.replace(' ', '')
+            new_key = key.replace('--jp-', '').replace('-', '_')
+            new_input_data[new_key] = new_value
+
+
+        with open('style/template.css', 'r') as file:
+            template = file.read()
```

Comment:
You can use modern Python for this:

```suggestion
      template = Path('style/template.css').read_text()
```

`Path` comes from the standard `pathlib`.